tidext/src/keyring/stronghold.rs

```diff
@@ -277,25 +284,34 @@ pub async fn init_stronghold_from_path<P: AsRef<Path>, T: AsRef<Vec<u8>>>(
 
 /// Try to get signer details for an existing stronghold instance at the specific location
 pub async fn get_signer_from_stronghold<T>(
```

Comment:
This no longer needs to be async right?

Reply:
No, doesn't need to be async.  I think I left it because it was just how the API was before.